src/xunit.performance.analysis/Program.cs

```diff
@@ -14,27 +14,25 @@
 using System.Xml.Linq;
 #if !LINUX_BUILD
 using System.Runtime.Serialization;
+using System.Xml;
 #endif
 
 namespace Microsoft.Xunit.Performance.Analysis
 {
     internal class Program
-    {
-        private const double ErrorConfidence = 0.95; // TODO: make configurable
-
+    {        
         /// <summary>
-        /// The name of the Duration metric, as provided by the XML.
+        /// The set of metrics (name of the metric) that appear in the .etl files
         /// </summary>
-        private const string DurationMetricName = "Duration";
-
+        
         private static int Usage()
         {
             Console.Error.WriteLine(
                 "usage: xunit.performance.analysis <xmlPaths> [-compare \"baselineRunId\" \"comparisonRunId\"]  [-xml <output.xml>] [-html <output.html>]");
             return 1;
         }
 
-        private static int Main(string[] args)
+        private static int Main( string[ ] args )
```

Comment:
You appear to be using custom spacing options. I suggest resetting to the Visual Studio defaults for C#.